back/spec/policies/project_policy_spec.rb

```diff
@@ -62,7 +62,7 @@
       it { is_expected.to permit(:index_xlsx) }
       it { is_expected.to permit(:survey_results) }
       it { is_expected.to permit(:submission_count) }
-      it { is_expected.to permit(:delete_inputs) }
+      it { is_expected.not_to permit(:delete_inputs) }
```

Comment:
Why did this change?

Reply:
On a project delete_inputs was only allowed if it was a continuous project. Removed that condition from the policy and now it always returns false